qa-test-app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
This file can be deleted completely to maintain consistency will all the other modules